### PR TITLE
Individual addressable left- and right screens

### PR DESCRIPTION
**Pull request by photon1503**
This is related to #20 

In order to have the option to show a dedicated screen (left- or right), it was necessary to change the properties in the dashboard.
The "Screen role" had to be removed and "Conditional overlay layer" enabed together with a proper condition.

image

Furthermore, all screens have now a black opaque background to avoid having them on the screen at the same time.

In the plugin I created a new class (Screens) and some properties.

**Breaking change**
The existing ActionA/B/C/D triggers are not working anymore.
Instead, new controls have to be bound.

image

There is a new "Dasboard" Menu to collect all Dasboard related things in one page.
(2) - New property to define a default starting page
(3) - Bindings for navigation through the screens

### CODE REVIEW

File: Sections/Dashboard.cs [hunk 1]
```diff
@@ -1,18 +1,25 @@
...
         private readonly DahlDesign Base;
 
- 
+        public Screen LeftScreen;
+        public Screen RightScreen;
         public Dashboard(DahlDesign dahlDesign)
         {
             Base = dahlDesign;
 
+            LeftScreen = new Screen(new string[] { "TimeAttack", "Sector", "LapTime", "Quali", "Race", "Race2", "Race3", "ShiftPoint", "Map" });
```
Comment (andreasdahl1987) [inline Sections/Dashboard.cs:16]: We should change the name of these pages:
- Time1
- Time2
- Time3
- Qualy
- Race1
- Race2
- Race3
- Practice1
- Practice2

File: Sections/Dashboard.cs [hunk 2]
```diff
@@ -1,18 +1,25 @@
...
+            RightScreen = new Screen(new string[] { "Practice", "Stint", "Q1", "Q2", "Race1", "Race2", "Ralley", "Track" });
```
Comment (andreasdahl1987) [inline Sections/Dashboard.cs:17]: We should change the name of these pages:
- Stint1
- Stint2
- Qualy1
- Qualy2
- Race1
- Race2
- Race3
- Track

File: Sections/Screen.cs [hunk 3]
```diff
@@ -0,0 +1,41 @@
+namespace DahlDesign.Plugin.Categories
+{
+    public class Screen
```
Comment (andreasdahl1987) [inline Sections/Screen.cs:3]: This is super cool :D

**Review by andreasdahl1987 — COMMENTED** [review]
@photon1503 Made some remarks

**Comment by andreasdahl1987** [comment]
Will test this in the evening :)

**Comment by photon1503** [comment]
Will change the names as suggested 
Also added actions to trigger every single page

The setup page is now cluttered but this should be cleaned up in a seperate process

**Comment by photon1503** [comment]
@andreasdahl1987 
done

we just need to do something with the layout. It's now too short to show all elements.

image

**Comment by photon1503** [comment]
@andreasdahl1987 - if you wonder who the old man from Discord is, thats me ;-)

**Pull request merged**